packages/o11ytsdb/rust/src/lib.rs

```
@@ -34,2456 +34,37 @@
 // No std, no allocator beyond a static scratch buffer.
 // All bit manipulation is native u64 — no BigInt overhead.
 
-#![no_std]
+#![cfg_attr(not(kani), no_std)]
```

Clippy (1.94.1)

[warning] unexpected `cfg` condition name

(warning)